### PR TITLE
Adds stand-alone usage to documentation

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Enhancement        | yes

### CODE REVIEW

File: docs/book/application-integration/stand-alone.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        return $renderer;
+    }
+);
+```
+
+#### Create View Script
+
+[Create a view script](../view-scripts.md) in `templates/index.phtml`:
+
+```php
```
Comment (michalbundyra) [inline docs/book/application-integration/stand-alone.md]: should it be php+html, not sure if it is supported

Comment (froschdesign) [reply docs/book/application-integration/stand-alone.md]: This results in wrong classes: `+html language-php`

**Comment by froschdesign** [comment]
@michalbundyra 
I updated the HTML code to reduce complexity and to support the display on GitHub:

https://github.com/laminas/laminas-view/blob/889765bf6339f704ae3dc97748ef740e23b6d851/docs/book/application-integration/stand-alone.md#create-view-model-and-render-output

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Pull request merged**